download_models/getModels.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+REFRESH_INPUT=
+OPEN_OVMS=0
+
+show_help() {
+    echo "
+        usage: $0
+            --workload dlstreamer|opencv-ovms
+            --refresh 
+        Note:
+            --refresh is optional, it removes previously downloaded model files
```

Comment:
if --workload is not required (i.e. optional) please make a note on that too and also states that it will use dlstreamer as the default value